Kitodo/src/test/java/org/kitodo/production/services/file/TreeDeleter.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.production.services.file;
```

Comment:
Please move this file to "org.kitodo" like the other Util-files in test.
In the specific test-directorys are only testfiles for the corresponding packages.

Reply:
Moved.